### PR TITLE
Prepare snapshot version 0.2.0

### PR DESCRIPTION
**Pull request by francabot**
Automated pull request for bump to version 0.2.0-SNAPSHOT.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,146 @@
 # Changelog
+ 
+## [v0.1.0](https://github.com/lf-lang/lingua-franca/tree/v0.1.0) (2022-04-04)
```
Comment (cmnrd) [inline CHANGELOG.md:3]: The changelog refers to version 0.1.0

File: gradle.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [header]
 group=org.lflang
-version=0.1.0-beta-SNAPSHOT
+version=0.1.0
```
Comment (cmnrd) [inline gradle.properties:3]: No bump to 0.2.0

File: org.lflang.lfc/src/org/lflang/lfc/Main.java [hunk 3]
```diff
@@ -52,7 +52,7 @@
 public class Main {
 
     /// current lfc version as printed by --version
-    private static final String VERSION = "0.1.0-beta";
+    private static final String VERSION = "0.1.0";
```
Comment (cmnrd) [inline org.lflang.lfc/src/org/lflang/lfc/Main.java:55]: No bump to 0.2.0

File: org.lflang.product/plugin.xml [hunk 4]
```diff
@@ -34,7 +34,7 @@
          </property>
          <property
                name="aboutText"
-               value="Version: 0.1.0-beta.&#x0A;&#x0A;Epoch is an IDE for Lingua Franca, a  polyglot coordination language for building concurrent and distributed software that delivers predictable and reproducible reactive behavior.&#x0A;&#x0A;Visit our GitHub: https://repo.lf-lang.org/.&#x0A;&#x0A;This software is licensed under the BSD 2-Clause License.&#x0A;Copyright © Lingua Franca contributors 2019-2021. All rights reserved.">
+               value="Version: 0.1.0.&#x0A;&#x0A;Epoch is an IDE for Lingua Franca, a  polyglot coordination language for building concurrent and distributed software that delivers predictable and reproducible reactive behavior.&#x0A;&#x0A;Visit our GitHub: https://repo.lf-lang.org/.&#x0A;&#x0A;This software is licensed under the BSD 2-Clause License.&#x0A;Copyright © Lingua Franca contributors 2019-2021. All rights reserved.">
```
Comment (cmnrd) [inline org.lflang.product/plugin.xml:37]: No bump to 0.2.0

**Review by cmnrd — COMMENTED** [review]
This PR seems to include changes for version 0.1.0-beta and then adds the changes for 0.2.0. Is this on purpose?

There seem to be a few places where the version has not been updated (see comments). Also the `splash.bmp` file shows version 0.1.0 (I cannot comment on the change unfortunately).

File: CHANGELOG.md (hunk 1 above)
Comment (lhstrh) [reply CHANGELOG.md:3]: Right. This is as it should be. The changelog only includes changes that have been released.

**Pull request merged**